dotcom-rendering/src/lib/discussionApi.tsx

```diff
@@ -436,3 +439,35 @@ export const getMoreResponses = async (
 
 	return parseCommentRepliesResponse(jsonResult.value);
 };
+
+export const getCommentContext = async (
+	ajaxUrl: string,
+	commentId: number,
+	filters: FilterOptions,
+): Promise<Result<GetDiscussionError, CommentContextType>> => {
+	const url = joinUrl(ajaxUrl, 'comment', commentId.toString(), 'context');
+	const params = buildParams(filters);
+
+	const response = await fetch(url + '?' + params.toString());
```

Comment:
If you look at other methods in this file you might be tempted to use `fetchJSON` which handles which returns a `Result`